### PR TITLE
[SP-6061] sending sdk errors to the server only in production (#320)

### PR DESCRIPTION
**Pull request by DrBaBo**
* added logger
https://www.facebook.com/profile.php?id=100082865407421
* added utility for debug

* fix tests

Co-authored-by: ciriti <ciriti@gmail.com>

### CODE REVIEW

**Comment by DrBaBo** [comment]
Hik

File: cmplibrary/src/debug/java/com/sourcepoint/gdpr_cmplibrary/exception/LoggerImpl.kt [hunk 1]
```diff
@@ -0,0 +1,66 @@
+@file:JvmName("LoggerFactory")
```
Comment (DrBaBo) [inline cmplibrary/src/debug/java/com/sourcepoint/gdpr_cmplibrary/exception/LoggerImpl.kt:1]: https://www.facebook.com/profile.php?id=100082865407421

File: cmplibrary/src/debug/java/com/sourcepoint/gdpr_cmplibrary/exception/LoggerImpl.kt [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+package com.sourcepoint.gdpr_cmplibrary.exception
+
+import android.util.Log
```
Comment (DrBaBo) [inline cmplibrary/src/debug/java/com/sourcepoint/gdpr_cmplibrary/exception/LoggerImpl.kt:5]: https://www.facebook.com/profile.php?id=100082865407421

**Review by DrBaBo — COMMENTED** [review]
I want login this account

File: cmplibrary/src/debug/java/com/sourcepoint/gdpr_cmplibrary/exception/LoggerImpl.kt (hunk 2 above)
Comment (DrBaBo) [reply cmplibrary/src/debug/java/com/sourcepoint/gdpr_cmplibrary/exception/LoggerImpl.kt:5]: +5

**Comment by andresilveirah** [comment]
@DrBaBo what's the purpose of this PR?

**Comment by carmelo-iriti** [comment]
@DrBaBo  I don’t understand this link https://www.facebook.com/profile.php?id=100082865407421 in the description

**Comment by carmelo-iriti** [comment]
The SP-6061 is already done

**Pull request closed without merging**